web/src/components/application-detail-page/application-detail/index.tsx

```diff
@@ -125,7 +125,30 @@ const MostRecentlySuccessfulDeployment: FC<{
             label="Deployed At"
             value={<span title={date.format()}>{date.fromNow()}</span>}
           />
-          <DetailTableRow label="Version" value={deployment.version} />
+          <DetailTableRow
+            label="Version"
+            value={
+              deployment.versionsList.length !== 0 ? (
+                <>
+                  {deployment.versionsList.map((v, i) => (
+                    <>
+                      {i > 0 && ","}
+                      <Link
+                        href={`//${v.url}`}
```

Comment:
should only add this `//` in case the url does not contain schema.
Can use v.url.includes(`://`) as the condition.